config/mail.php

```diff
@@ -38,6 +38,7 @@
     'mailers' => [
 
         'smtp' => [
+            'scheme' => env('MAIL_MAILER', 'smtp'),
```

Comment:
This is incorrect, possible values of `MAIL_MAILER` are "smtp", "sendmail", "mailgun", "ses", "ses-v2", "postmark", "resend", "log", "array", "failover" or "roundrobin"

Reply:
Yes, you are right this should be of course a new variable, my mistake I added our hotfix instead of the generic config option.